tools/scripts/update-bumpenvs-yaml.py

```diff
@@ -39,6 +42,7 @@
     "build-and-publish-docker-tf1-cpu",
     "build-and-publish-docker-tf1-gpu",
     "build-and-publish-docker-tf2-gpu",
+    "build-and-publish-docker-cuda-11",
```

Comment:
I am not familiar with the cuda 11 upgrading. Aren't planning to upgrade all the images to have cuda 11? Is this image a separate image that only contains cuda 11? If so, why do we want to do that?

Reply:
So we don't want all images to be built with CUDA 11 (yet) because that would require anyone still using CUDA 10 drivers to stop. We do want to enable A100s in the next release, so there will be an image built with CUDA 11 and equivalent versions of all the frameworks. That's the main reason for the move to TF 2.4 and PT 1.7 (although there's a couple of other features, and it's about time to upgrade anyway). So this commit is focused on move to the new images that have parity with the required framework versions. As soon as it's in, I have some CI changes that will exercise the CUDA 11 images on a regular basis with all of our GPU tests.